recipes/erikzenker-hsm/all/conanfile.py

```diff
@@ -1,51 +1,71 @@
-from conan import ConanFile, tools
-from conan.tools.cmake import CMake, cmake_layout
+from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
 from conan.tools.scm import Version
 import os
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.50.0"
+
 
 class HsmConan(ConanFile):
     name = "erikzenker-hsm"
     license = "MIT"
     homepage = "https://github.com/erikzenker/hsm.git"
     url = "https://github.com/conan-io/conan-center-index"
-    description = "The hana state machine (hsm) is a finite state machine library based on the boost hana meta programming library. It follows the principles of the boost msm and boost sml libraries, but tries to reduce own complex meta programming code to a minimum."
+    description = (
+        "The hana state machine (hsm) is a finite state machine library based "
+        "on the boost hana meta programming library. It follows the principles "
+        "of the boost msm and boost sml libraries, but tries to reduce own "
+        "complex meta programming code to a minimum."
+    )
     topics = ("state-machine", "template-meta-programming")
-    requires = "boost/1.79.0"
-    no_copy_source = True
-    settings = "os", "arch", "build_type", "compiler"
-    generators = "CMakeDeps", "CMakeToolchain"
 
-    def layout(self):
-        cmake_layout(self)
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "CMakeDeps"
+
+    def requirements(self):
+        self.requires("boost/1.79.0")
+
+    def package_id(self):
+        self.info.header_only()
```

Comment:
```suggestion
        self.info.clear()
```

Wouldn't it be better to use `clear()`?

https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#the-package-id-method

Reply:
Yes 👍